packages/angular/common/src/directives/navigation/router-link-delegate.ts

```diff
@@ -31,12 +31,45 @@ export class RouterLinkDelegateDirective implements OnInit, OnChanges {
 
   ngOnInit(): void {
     this.updateTargetUrlAndHref();
+    this.updateTabindex();
   }
 
   ngOnChanges(): void {
     this.updateTargetUrlAndHref();
   }
 
+  /**
+   * The `tabindex` is set to `0` by default on the host element when
+   * the `routerLink` directive is used. This causes issues with Ionic
+   * components that wrap an `a` or `button` element, such as `ion-item`.
+   * See issue https://github.com/angular/angular/issues/28345
+   *
+   * This method removes the `tabindex` attribute from the host element
+   * to allow the Ionic component to manage the focus state correctly.
+   */
+  private updateTabindex() {
+    // Ionic components that render a native anchor or button element
+    const ionicComponents = [
+      'ION-BACK-BUTTON',
+      'ION-BREADCRUMB',
```

Comment:
When testing it on standalone, the `ion-breadcrumb` becomes a string when using `routerLink` instead. This happens before the fix so I don't think it's related. If this isn't supposed to be happening, then we should make a new ticket to investigate this.

Reply:
I created ticket `ROU-10993` to address this. I tested it out on `main` and the same issue occurs so I don't think it should block this PR.